precompiles/staking/StakingI.sol

```diff
@@ -238,6 +268,26 @@ interface StakingI is authorization.AuthorizationI {
             PageResponse calldata pageResponse
         );
 
+    /// @dev CreateValidator defines an Event emitted when a create a new validator.
+    /// @param delegatorAddress The address of the delegator
+    /// @param validatorAddress The address of the validator
+    /// @param commissionRate The commission rate charged to delegators, as a fraction
+    /// @param commissionMaxRate The commission max rate charged to delegators, as a fraction
+    /// @param commissionMaxChangeRate The commission max change rate charged to delegators, as a fraction
+    /// @param minSelfDelegation The validator's self declared minimum self delegation
+    /// @param pubkey The consensus public key of the validator
+    /// @param value The amount of coin being self delegated
+    event CreateValidator(
+        address indexed delegatorAddress,
+        string indexed validatorAddress,
```

Comment:
```suggestion
        address indexed validatorAddress,
```

Reply:
To be honest, I also think that using type **address** is more reasonable than **string**. But I saw that your other events all use **string indexed validatorAddress**. I think should keep it consistent.

Reply:
done !

Reply:
@luchenqun lets use address. @Vvaradinov just added the bech32 precompile to help out with this

Reply:
Also string addresses are not correctly indexed

Reply:
The string is indeed not indexed correctly. In fact, I am also very confused. I just wrote it like this to be consistent with the previous code.